packages/contracts-ops/contracts/scripts/UpgradeCallBatches.s.sol

```diff
@@ -16,41 +16,11 @@ import {CallBatch} from "../CallBatch.sol";
 // Contract for selector
 import {UpgradeBeaconController} from "@nomad-xyz/contracts-core/contracts/upgrade/UpgradeBeaconController.sol";
 
-contract UpgradeCallBatches is Script, Config, CallBatch {
+contract UpgradeCallBatchLogic is Script, Config, CallBatch {
     string currentDomain;
 
-    function printCallBatches(
-        string memory _configFile,
-        string[] memory _domainNames,
-        string memory _localDomainName,
-        bool recovery
-    ) external {
-        localDomainName = _localDomainName;
-        setUp(_configFile);
-        for (uint256 i; i < _domainNames.length; i++) {
-            currentDomain = _domainNames[i];
-            generateGovernanceCalls();
-        }
-        writeCallBatch(recovery);
-    }
-
-    function setUp(string memory _configFile) internal {
-        __Config_initialize(_configFile);
-        string memory outputFile = "upgradeActions.json";
-        __CallBatch_initialize(
-            localDomainName,
-            getDomainNumber(localDomainName),
-            outputFile,
-            true
-        );
-        console2.log("Governance Actions have been output to", outputFile);
-    }
-
-    /*//////////////////////////////////////////////////////////////
-                       GOVERNANCE CALL GENERATORS
-    //////////////////////////////////////////////////////////////*/
-
-    function generateGovernanceCalls() internal {
+    function upgrade(string memory _domain) internal {
+        currentDomain = _domain;
```

Comment:
yeah, agree. it's the convention we had used for other scripts generating actions (`enrollReplica`, `rotateUpdater`, etc) but i think we could rename to something more clearly accurate, though it would be more verbose

Reply:
do you think `pushUpgrade`, `generateUpgrade`, something else ??

Reply:
I think that these names are weird as well. For someone who is not very familiar to the codebase, it's not intuitive that the actions are actually executed via the batch system. Even I had to doublecheck my understanding with James after seeing these names :P

Reply:
yeah sigh, the governance system is not super easy to grok for someone else, cuz obv we aren't just taking action directly, we are basically constructing them and queueing them up to be taken by a Higher Power

Reply:
i went with push.. https://github.com/nomad-xyz/monorepo/pull/511/commits/29cef19fa75a76d8ae45a6fc2aad7f912f69a1da